app/src/main/java/org/sinisterstuf/guesstheanimal/Game.java

```diff
@@ -1,13 +1,6 @@
 package org.sinisterstuf.guesstheanimal;
 
-import android.app.Activity;
-import android.os.Bundle;
-import android.content.Intent;
-import android.view.Menu;
-import android.view.View;
```

Comment:
This file was introduced in d66c1f2, probably with the intent of being the main file, but it looks like I didn't follow through.

Reply:
Yeah, I just meant this as a historical comment as to explain how a one-line file ended up here with so many imports :smile: